src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+  font-style: normal;
+  font-weight: 500;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  padding-right: 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding-inline-start: 0;
```

Comment:
why do you need such a weird `padding`?

Reply:
because for "ul" there is default padding-inline-start: 40px

Reply:
it's better to use just `padding: 0;`)